File: programacao/Ex18.c
#include <stdio.h>
#include <stdlib.h>  

int main(int argc, char *argv[])
{
  float peso, altura, imc;
  char sexo;

   printf("Digite o sexo(M ou F): " );
  scanf("%s",&sexo);
  printf("Digite o seu peso em Kg: ");
  scanf("%f", &peso);
  printf("Digite a sua altura em M: ");
  scanf("%f", &altura);
  imc = peso / (altura*altura);
 if(sexo ==  'F' )
 {
    if(imc<19)
    {
      printf("Abaixo do peso" );
    }  
    else if(imc>=19 && imc <24)
     {
      printf("Peso ideal" );
    }
    else
    {
      printf("Acima do peso" );
    }  
    printf("\n seu imc e %.2f", imc);
 }

 else if(sexo ==  'M' )
  {
    if(imc<20)
    {
      printf("Abaixo do peso" );
    }  
    else if(imc>=20 && imc <25) 
    {
      printf("Peso ideal" );
    }
    else
    {
      printf("Acima do peso" );
    }  
    printf("\n seu imc e %.2f", imc);
  }

      
  return 0;
}

File: python/Ex18.py
sexo = float(input("Digite F: para sexo feminino, qualquer coisa para sexo Masculino:"))
peso = float(input("Digite o peso em kg: "))
altura = float(input("Digite o altura em m: "))
IMC = peso / altura**2
if(sexo == 1):
    IMC = IMC + 1
if(IMC < 20):
    print("Pessoa esta abaixo do peso")
else:
    if(IMC < 25):
        print("Pessoa tem peso normal")
    else:
        if(IMC < 30):
            print("Pessoa esta acima do peso")
        else:
            print("Pessoa esta Obesa")
